agent/ecscni/types.go

```diff
@@ -124,6 +131,28 @@ type ENIConfig struct {
 	SubnetGatewayIPV4Address string `json:"subnetgateway-ipv4-address"`
 }
 
+// AppMeshConfig contains all the information needed to invoke the app mesh plugin
+type AppMeshConfig struct {
+	// Type is the cni plugin name
+	Type string `json:"type,omitempty"`
+	// CNIVersion is the cni spec version to use
+	CNIVersion string `json:"cniVersion,omitempty"`
+	// Egress traffic from the processes owned by the UID will be ignored
+	IgnoredUID string `json:"ignored-uid,omitempty"`
```

Comment:
For cni configuration, normally it would follow camel case. That is also what I implemented in cni plugin. These names need to be updated.

Reply:
sounds good, updated